src/ActionRow/index.jsx

```diff
@@ -1,22 +1,47 @@
-import React from 'react';
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
+
+const styles = {
+  horizontal: css`
+    display: flex;
+    flex-grow: 1;
+    align-items: center;
+    justify-content: flex-end;
+    & > * {
+      margin: 0;
+    }
+    & > * + * {
+      margin-left: .5rem;
+    }
+  `,
+  stacked: css`
+    display: flex;
+    flex-grow: 1;
+    align-items: center;
+    flex-direction: column-reverse;
+    justify-content: center;
+    & > * {
+      margin: 0;
+    }
+    & > * + * {
+      margin-bottom: .5rem;
+    }
+  `,
+};
 
 const ActionRow = ({
-  as,
+  as: Component,
   isStacked,
   children,
   ...props
-}) => React.createElement(
-  as,
-  {
-    ...props,
-    className: classNames(props.className, {
-      'pgn__action-row': !isStacked,
-      'pgn__action-row-stacked': isStacked,
-    }),
-  },
-  children,
+}) => (
+  <Component
+    css={isStacked ? styles.stacked : styles.horizontal}
```

Comment:
Do you have any opinions on usage of `css` vs. using the `cx` function with `className` here?

The latter would provide an API more like the `classnames` package, letting us put the common properties together here on line 40, and use the conditional to add on the ones that are different.

I'm just reading and learning, not strongly opinionated about it yet. 😉

Reply:
I was just thinking that would make sense to do